WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -213,14 +214,26 @@ public void onClick(View v) {
                                         }
                                     }).show();
                         } else if (savedLocally) {
-                            Snackbar.make(getActivity().findViewById(R.id.coordinator),
-                                    R.string.editor_post_saved_locally_not_published, Snackbar.LENGTH_LONG)
-                                    .setAction(R.string.button_publish, new View.OnClickListener() {
-                                        @Override
-                                        public void onClick(View v) {
-                                            publishPost(post);
-                                        }
-                                    }).show();
+                            if (isScheduledPost) {
```

Comment:
I think we could just set two variables for the strings here rather than duplicating this large block.

```
String text = isScheduledPost ? R.string.editor_scheduled_post_saved_locally : R.string.editor_post_saved_locally_not_published;
String buttonText = isScheduledPost ? R.string.button_save : R.string.button_publish;
...
```